rootfs/bin/doctor

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -eo pipefail
+uid=`curl --silent --show-error  -X POST -d "skip" localhost:8080/doctor`
+echo "Information sent to deis doctor is available at the follwoing url  https://doctor-staging.deis.com/${uid}"
```

Comment:
Let's replace the literal string URL `https://doctor-staging.deis.com/${uid}` with `${DOCTOR_API_URL}/${uid}`

(and there's a spelling error: `s/follwoing/following`)

(PS I still think we should omit `-d "skip"`)